packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -957,6 +957,11 @@ export const Table = <
                           (c) => c.id !== SELECTION_CELL_ID, // first non-selection column is the expander column
                         );
 
+                    // expander column should be wider to accommodate the expander icon
+                    if (columnHasExpanders && column.minWidth === 72) {
+                      column.minWidth = 108;
+                    }
```

Comment:
Would it be better to have something like an object with the possible widths instead of directly using the hardcoded numbers in code? E.g.

```jsx
const defaultColumnWidths = { 
  default: 72,
  widthExpander: 108
};

…

// expander column should be wider to accommodate the expander icon
if (columnHasExpanders && column.minWidth === defaultColumnWidths.default) {
  column.minWidth = defaultColumnWidths.withExpander;
}
```

Reply:
Good suggestion, done: [`f44a13b`](https://github.com/iTwin/iTwinUI/pull/1863/commits/f44a13b0f2ccb3e668ea18699231622b7c32f1ec)